web/src/components/tasks/cannabis/CannabisPriorityTypes.tsx

```diff
@@ -53,6 +55,71 @@ export const CannabisPriorityTypes = (props: Props): ReactElement => {
     }
   }, [userData, update]);
 
+  useEffect(() => {
+    if (!userData) return;
+    const anyMinorityOrWomenCheckboxesSelected: boolean = priorityTypesObj.minorityOrWomen.some(
```

Comment:
This logic all works great, but it could be consolidated. 
Instead of doing the check, storing it as a variable and then using that variable in the if block, you could have the if block be the check. 

This would make it more clear and concise. 

That said, I don't think that needs to be changed before this ticket is merged to main. It could be done as a refactoring later.

Reply:
Same as my comment :) your idea for making it more concise is actually different from mine. it might be fun to do a refactoring party of sorts. to play with different possible refactorings and teach/show that there's never exactly one "correct" answer or "best" solution